haptic_mpc:
  pr2:
    # Skin topic lists. Each should be publishing a TaxelArray message
    skin_list:
      none: []
      fabric: []    
      pps: []
      bosch: []

    # Frames used for TF. torso_frame is the one the arm is attached to, inertial frame is to get the torso position in the world
    torso_frame: 'torso_lift_link'
    inertial_frame: 'base_link'
    end_effector_frame: 'r_gripper_tool_frame'

    # Inposition parameter for action and log
    ee_motion_threshold: 0.01
    ee_orient_motion_threshold: 0.01
    jts_motion_threshold: 0.01

    # Joint limits.
    # Should be equal in number and in the same order as the number of links in the arm (eg 7 for the PR2, starting at the shoulder). 
    # Should be specified in DEGREES.
    joint_limits:
        max: [26.0, 68.0, 41.0, 0.010, 270.0, 0.01, 180.0]
        min: [-109.0, -24.0, -220.0, -132.0, -270.0, -120.0, -180.0]

    # tool frame  
    tool_id: 0

  groups:
    - name: right_arm_joints
      joints:
        - r_shoulder_pan_joint
        - r_shoulder_lift_joint
        - r_upper_arm_roll_joint
        - r_elbow_flex_joint
        - r_forearm_roll_joint
        - r_wrist_flex_joint
        - r_wrist_roll_joint
    - name: right_arm_links
        - r_upper_arm_link
        - r_elbow_flex_link
        - r_forearm_link
        - r_wrist_flex_link
        - r_gripper_palm_link
        - r_gripper_l_finger_link
        - r_gripper_l_finger_tip_link
        - r_gripper_r_finger_link
        - r_gripper_r_finger_tip_link
    - name: right_arm
      base_link: torso_lift_link
      tip_link: r_gripper_tool_frame



